tests/test_interface.sh

```diff
@@ -80,5 +80,89 @@ if output=$(TESTING_SKIP_ROOT_CHECK=1 "$CC_SNAPSHOT" -d mytest 2>&1); then
 else
   fail "Dry-run exited with error: $output"
 fi
+# Srart of custom source path tests
+
+export CC_SNAPSHOT_TAR_PATH="${PWD}/snapshot.tar"
+DEFAULT_TAR="${PWD}/snapshot.tar"
+TESTDIR="/tmp/cc_test"
```

Comment:
maybe generate a unique name under here in case this exists from a previous run on a system?